frappe/public/scss/common/datepicker.scss

```diff
@@ -5,7 +5,7 @@
 .datepicker {
 	direction: ltr;
 	font-family: inherit;
-	z-index: 9999 !important;
```

Comment:
Why reduce the z-index?

Reply:
Because datepicker open above the header.

Reply:
![image](https://github.com/ParaLogicTech/frappe/assets/139151580/d3702f92-0667-4dfe-ac6d-3dd7ef006193)